### ISSUE TITLE
Skip/mute YouTube ads? 

### ISSUE THREAD

**Issue by ryankupk** [dmunozv04/iSponsorBlockTV#149]
The readme mentions the following, and as far as I can tell, it's not mentioned anywhere else

> It can also skip/mute YouTube ads.

How do you do this? I'm happy to update some docs if I understand how to do this. It does not appear to happen by default as I've set up sponsorblock and it works fine on my Samsung TV, but regular youtube ads aren't muted or automatically skipped

**Comment by ryankupk** [comment]
I checked the config.json file in the data directory and switched `mute_ads` and `skip_ads` from false to true, did some quick testing and actually didn't get any skippable ads but the ads I did get were muted, so seems like that fixes it. But, should this be part of the config wizard? I set this up with the cli and there wasn't a prompt for muting or skipping ads. Should this issue be an enhancement to add that? I'm not sure if it works in the textual wizard as I wasn't able to get that to work but maybe there just needs to be parity between the two if so, or be added to both. Unless I'm missing something entirely

**Comment by skupjoe** [comment]
I thought that this wasn't working either until I checked the conf and set these to `true` as well.

This should be part of the CLI setup!

**Comment by dmunozv04** [comment]
@ryankupk thanks for closing the issue. I’ll release a new version in the near future so this change is on the ‘latest’ tag on docker and on pypi

**Comment by ryankupk** [comment]
Thanks @dmunozv04, I hope I didn't mess up your workflow or anything. I just came back to the repo because I was going to raise a PR for the minimum skip segment, saw that there was a PR for it already so dropped a comment referencing this issue/PR, saw that this was still open so I closed it since the PR was merged. I didn't know if you were planning to close after making a new release or something so apologies if me closing it was annoying.
